src/java.desktop/share/classes/java/beans/Beans.java

```diff
@@ -64,6 +64,22 @@ public Beans() {}
      * <p>
      * Instantiate a JavaBean.
      * </p>
+     * The bean is created based on a name relative to a class-loader.
+     * This name should be a dot-separated name such as "a.b.c".
+     * <p>
+     * The given name can indicate either a serialized object or a class.
+     * We first try to treat the beanName as a serialized object
```

Comment:
```suggestion
     * We first try to treat the {@code beanName} as a serialized object
```
Should this be marked up as it refers the parameter name? Also, in other instances.

Reply:
done